### ISSUE TITLE
Automatic update to 0.11.0 stops after upgrading master node

### ISSUE THREAD

**Issue by jkovacs-dev** [rancher/k3os#556]
**Version (k3OS / kernel)**
k3os version v0.11.0

**Architecture**
x86_64

**Describe the bug**
After upgrading my master node to 0.11.0, it appears that the matching rule on k3os-latest plan has been modified to look for the label `k3os.io/upgrade` instead of `plan.upgrade.cattle.io/k3os-latest` as indicated in the documentation.  

**To Reproduce**

1. Create a 2 node cluster with master and worker nodes running k3os 0.10.0.
2. Run `kubectl label nodes -l k3os.io/mode plan.upgrade.cattle.io/k3os-latest=enabled`
3. Wait for master node to upgrade to k3os release 0.11.0.

**Expected behavior**
All nodes to upgrade to latest release.

**Actual behavior**
Only the master node gets upgraded.  No upgrade job is scheduled for the worker node.  
```
NAME          STATUS   ROLES    AGE   VERSION
k3os-agent1   Ready    <none>   11d   v1.18.4+k3s1
k3os-master   Ready    master   11d   v1.18.6+k3s1
```
**Additional context**
Applying the label `k3os.io/upgrade=enabled` to the other nodes will kick off the upgrade as expected.  I suspect this change is intentional and the documentation just needs to be updated to reflect the new label.

**Comment by dweomer** [comment]
@jkovacs-dev Yep, my bad. This is in preparation for a k3s-upgrade style servers-before-agents dual plan that will be coming with the next upgrade.
